### DISCUSSION TITLE
Working RAM air turbine

### DISCUSSION THREAD

**Discussion by KBNOtaku** [magknight/787aviatorsTracker#535] [Ideas]
**Is your feature request related to a problem? Please describe.**
For emergancy situations pleas add an working RAM Turbin.

**Describe the solution you'd like**
A working RAM Turbin that supports the Aircraft with power.

**Comment by aeroniemi** [comment]
it's only used for emergency scenarios (and pretty dire ones at that) - no intention to add failure support in the near future, but may be open to it at some point in the future
